elyra/pipeline/registry.py

```diff
@@ -523,6 +523,17 @@ def parse_component_properties(self, component_body, component_path):
 
                     # Add to existing parameter list
                     component_parameters['parameters'].append({"id": new_parameter_info['parameter_ref']})
+                    if(default_value is not None and len(default_value) > 0):
+                        if 'data' in new_parameter_info:
```

Comment:
Should there be an immediate check against `default_value == 'None'` to set `default_value = None`, _then_ go through the data-type checks, etc.?